tools/nanopolish/nanopolish_methylation.xml

```diff
@@ -39,6 +40,8 @@
 
         <param argument="-w" type="text" optional="true"
             label="find variants in window of region chromsome:start-end" />
+        
+        <param name="batchsize" type="integer" optional="true" value="512" label="the batch size" help="(-K)"/>
```

Comment:
if you set optional=true, I would use value="" and include `#if str($batchsize)` in the command section.

Reply:
So you mean, it's preferred to not pass optional params with their default values?

Reply:
It should be fine, but 1) if the user removes everything then the value is "" and the command crashes, because you do not catch it and 2) if upstream changes the default value at some point you need to update the default value in your case. If you don't hardcode it, you don't need to change it :)